docs/platforms/javascript/common/session-replay/issue-types.mdx

```diff
@@ -0,0 +1,50 @@
+  - javascript.deno
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+description: "Learn about the Issue types that Session Replay can detect."
+---
+
+A _[replay issue](/product/issues/issue-details/replay-issues/)_ is an issue detected using captured Session Replay data. If your application is configured with [Session Replay](/product/session-replay/), problems will be detected on the server side during replay ingest, and grouped into issues. We group similar events into issues based on a fingerprint. For replay issues, a fingerprint is primarily based on the problem type and the url or transaction name where the problem occurred.
+
+You can configure which issue types are created by visiting Project Settings > Replay and toggling detection on or off for each issue type.
+
+![Replay settings page](./img/replay-settings.png)
+
+## Configuring Rage Click Issues
+
+"Slow clicks" (also called "dead clicks") are only detected on `<button>`, `<input>`, and `<a>` elements that don't lead to updates to the DOM or a page scroll within 7 seconds. When the user clicks on one of these elements 3 or more times within that 7 second time frame, it indicates frustration, and the SDK registers a ["rage click"](/product/issues/issue-details/replay-issues/rage-clicks/).
+
+Sometimes rage or dead clicks will be detected on elements which are not expected to mutate the DOM, like a "Print" or "Download" button. In this case, you can configure `slowClickIgnoreSelectors`, which will prevent those buttons from generating new issues. See <PlatformLink to="/session-replay/troubleshooting/#why-am-i-seeing-too-many-or-too-few-rage-clicks">Why am I seeing too many or too few rage clicks?</PlatformLink> for more.
+
+## Configuring Hydration Errors
+
+When a [hydration error](/product/issues/issue-details/replay-issues/hydration-error/) occurs in your React app, the SDK will emit an error object as well as a replay breadcrumb, both of which represent the problem. Both of these are sent to the server for processing.
+
+The error event, before considering available quota, is subject to [Inbound Filters](https://docs.sentry.io/concepts/data-management/filtering/) and may be dropped. If the inbound filters are disabled, the error is accepted (consuming error quota) and an issue is created. Error events may or may not have an attached replay to help debug the problem.
+
+The replay breadcrumb is stored as part of the replay session if the server has already accepted the replay. While storing the breadcrumb, the ingest pipeline looks at your project settings and will create a replay issue based on this hydration breadcrumb if the preference is enabled in Project Settings > Replay. Each event instance inside the issue will have an associated replay, to make debugging easier.
```

Comment:
I think we could leave out the detail of the breadcrumb. It's a bit too much detail given that this is rather complicated to explain

Reply:
rewriting this section to be shorter...